### DISCUSSION TITLE
Feature request: Tools -> DDEV -> Enable/Disable Xdebug

### DISCUSSION THREAD

**Discussion by srepn** [php-perfect/ddev-intellij-plugin#136] [Ideas]
Hello ! 
Thanks for the great tool, really love it.
Was thinking about an option for disable/enable Xdebug and observer  the current status from UI.
Thanks!

**Comment by nico-loeber** [comment]
Hey @sergiirepin,

You can configure ddev in a way you do not need to switch xdebug on and off for debugging. This would be the prefered approach in my eyes.

See:
* https://github.com/drud/ddev/issues/3731

This might be related to:
* https://github.com/php-perfect/ddev-intellij-plugin/discussions/110

**Reply by rfay** [reply]
I sure would love to get that more clearly documented or implemented, 
* https://github.com/drud/ddev/issues/3731#issuecomment-1412082405
